docs/quasar.conf.js

```diff
@@ -85,7 +85,11 @@ module.exports = function (ctx) {
         loadingBar: {
           color: 'amber'
         }
-      }
+      },
+
+      plugins: [
```

Comment:
Quasar is configured to import everything already. See `all: true` above. This is redundant.